eng/Build.ps1

```diff
@@ -226,6 +226,7 @@ function BuildSolution([string] $solutionName) {
         /p:QuietRestoreBinaryLog=$binaryLog `
         /p:TestTargetFrameworks=$testTargetFrameworks `
         /p:DotNetBuildFromSource=$sourceBuild `
+        /p:IsGraphBuild=$True `
```

Comment:
setting the property doesn't actually trigger the different build handling, you need to use the `/graph` flag for that instead. 
```suggestion
        /graph
```